spec.bs

```diff
@@ -493,10 +601,12 @@ To <dfn>evaluate script</dfn> with given [=string=] <dfn for="evaluate script">|
 
 This specification defines two [=policy-controlled features=] identified by the string
 "<code><dfn noexport>join-ad-interest-group</dfn></code>", and
-"<code><dfn noexport>run-ad-auction</dfn></code>". Their [=policy-controlled feature/default allowlists=] are `'self'`.
+"<code><dfn noexport>run-ad-auction</dfn></code>". Their
+[=policy-controlled feature/default allowlists=] are
+<a href="https://w3c.github.io/webappsec-permissions-policy/#default-allowlist">`self`</a>.

```

Comment:
this change is now in a separate [PR](https://github.com/WICG/turtledove/pull/487). Once that's merged, this PR will get this change from that.